ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs

```diff
@@ -100,6 +101,7 @@ private void LoadHistogramData(AlternativeComparisonReportResults altResults, in
             IHistogram histogram = altResults.GetAlternativeResultsHistogram(altID);
             double[] binValues = histogram.BinCounts.Select(i => (double)i).ToArray();
             _data = new HistogramData2D(histogram.BinWidth, 0, binValues, "Chart", "Series", "X Data", "YData");
```

Comment:
Just making a note that this should use a different X and Y axis ID.  I know you're on it, so just resolve this when you're done with it.

Reply:
I think this needs an updated min maybe?